tensorflow/core/common_runtime/gpu/pool_allocator_test.cc

```diff
@@ -24,9 +24,15 @@ limitations under the License.
 namespace tensorflow {
 namespace {
 
+#if GOOGLE_CUDA
+static const char* PlatformName = "cuda";
+#else
```

Comment:
#elif TENSORFLOW_USE_ROCM ?

Reply:
actually replace the local var with calls to this routine instead.

https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/blob/develop-upstream/tensorflow/core/common_runtime/gpu/gpu_init.cc#L45